packages/grafana-ui/src/components/GraphNG/GraphNG.story.tsx

```diff
@@ -1,15 +1,16 @@
 import { FieldColorModeId, toDataFrame, dateTime } from '@grafana/data';
 import React from 'react';
 import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
-import { GraphNG, GraphNGProps } from './GraphNG';
+import { GraphNGProps } from './GraphNG';
 import { LegendDisplayMode, LegendPlacement } from '../VizLegend/models.gen';
 import { prepDataForStorybook } from '../../utils/storybook/data';
 import { useTheme2 } from '../../themes';
 import { Story } from '@storybook/react';
+import { TimeSeries } from '../TimeSeries/TimeSeries';
 
 export default {
   title: 'Visualizations/GraphNG',
-  component: GraphNG,
+  component: TimeSeries,
```

Comment:
This is GraphNG story, i think you can maybe make it GraphNG back, or rename to TimeSeries ?

Reply:
Can we improve the naming in the next round?  I am pretty convinced one of these levels will go away, but for now the difference of TimeSeries vs GraphNG is X==time.

Reply:
this is nit, the story is not very maintained anyways so YES!